src/package_report.py

```diff
@@ -48,7 +56,9 @@ def _get_package_versions_in_upstream(target_packages_match_spec_out, target_ver
     return package_to_version_mapping
 
 
-def _generate_report(package_versions_in_upstream, target_packages_match_spec_out, image_config, version):
+def _generate_staleness_report_per_image(
+    package_versions_in_upstream, target_packages_match_spec_out, image_config, version
+):
     print("\n# Staleness Report: " + str(version) + "(" + image_config["image_type"] + ")\n")
```

Comment:
can we use create_markdown utility function here ?

Reply:
Yeah I think it is also possible to use it here, I'll do the refactor